### PR TITLE
chore: remove deprecated method

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

File: google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java [hunk 1]
```diff
@@ -66,14 +64,22 @@ public void testBuilder() {
 
   @Test
   public void testTypeOf_Npe() {
-    thrown.expect(NullPointerException.class);
-    TimePartitioning.of(null);
+    try {
+      TimePartitioning.of(null);
+      fail();
```
Comment (rahulKQL) [inline google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java]: It is more readable to keep the class name `Assert.fail()`

**Review by rahulKQL — APPROVED** [review]

File: google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java (hunk 1 above)
Comment (pmakani) [reply google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java]: @rahulKQL only fail or also assertNotNull ?

Comment (rahulKQL) [reply google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TimePartitioningTest.java]: Only for fail, assertNotNull is readable without classname

**Comment by rahulKQL** [comment]
LGTM

**Pull request closed without merging**